File: example/src/main/java/com/github/appintro/example/ui/mainTabs/CustomLayoutIntro.java
package com.github.appintro.example.ui.mainTabs;

import android.content.Intent;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.github.appintro.example.R;

import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

/**
 * Created by andrew on 11/17/16.
 */

public class CustomLayoutIntro extends Fragment {

    @SuppressWarnings("ConstantConditions")
    @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);

        Button demo = getView().findViewById(R.id.intro3);
        demo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(getActivity().getBaseContext(), com.github.appintro.example.ui.mainTabs.intro.CustomLayoutIntro.class));
            }
        });
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.tab_custom_layout_intro, container, false);
    }
}

File: example/src/main/java/com/github/appintro/example/ui/mainTabs/CustomLayoutIntro.kt
package com.github.appintro.example.ui.mainTabs

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.github.appintro.appintroexample.R
import com.github.appintro.example.ui.mainTabs.intro.CustomLayoutIntro
import kotlinx.android.synthetic.main.tab_custom_layout_intro.*

class CustomLayoutIntro : Fragment() {
    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        intro3.setOnClickListener {
            startActivity(Intent(context, CustomLayoutIntro::class.java))
        }
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? =
            inflater.inflate(R.layout.tab_custom_layout_intro, container, false)
}